### ISSUE TITLE
Deaf and signing issues: Size for signing videos [XAUR] 

### ISSUE THREAD

**Issue by RealJoshue108** [w3c/apa#185]
Based on our RQTF call today

https://www.w3.org/2021/05/26-rqtf-minutes.html

I note @raja-kushalnagar-gallaudet suggestion to mention the importance of size for any signing videos. Similar to the WCAG G54 Technique 'Including a sign language interpreter in the video stream'. This is a useful addition to the XAUR.

**Comment by RealJoshue108** [comment]
@raja-kushalnagar-gallaudet  Any links to related research appreciated

**Comment by raja-kushalnagar-gallaudet** [comment]
Here's my paper presentation at the 2018 Accessibility Testing Symposium
that recommended1/3rd minimum signing size.

The paper looked at 240 ASL videos intended to be watched by deaf signers,
and found that nearly all set signer size to at least one-third the size of
the full video.

**Comment by RealJoshue108** [comment]
Great stuff @raja-kushalnagar-gallaudet  I don't see the paper, have you a URI? thanks

**Comment by raja-kushalnagar-gallaudet** [comment]
I uploaded my ICT Accessibility Testing Symposium presentation paper to arxiv, and its URI will be available on Friday morning. Here is an alternate URI for this paper: [ASL_Legibility](https://www.csua.berkeley.edu/~rajah/publications/ASL_Legibility.pdf)

**Comment by RealJoshue108** [comment]
Perfect! Do forwrad thanks @raja-kushalnagar-gallaudet

**Comment by RealJoshue108** [comment]
@raja-kushalnagar-gallaudet  I've added a new draft requirement based on your suggestion to this branch. We will review in Research Questions next week. Appreciated, I think this is a great addition - any comments or suggestions please let me know. Thanks

http://raw.githack.com/w3c/apa/xaur-signing-avatar/xaur/index.html

**Comment by RealJoshue108** [comment]
Discussed at RQTF 26th May meeting https://www.w3.org/2021/05/26-rqtf-minutes.html

**Comment by JWJPaton** [comment]
Hi there,
Late to the discussion but some deafblind sign language users also have tunnel vision (some forms of USHER syndrome) so for them there is a maximum usable size as well. I don't have any specifics in terms of sizes though I'm afraid.

@raja-kushalnagar-gallaudet did your research include deafblind signers? 

Thanks

**Comment by raja-kushalnagar-gallaudet** [comment]
We have not conducted studies on DB caption access, and we will try to do
that in the near future.

Related to your question -- we had conducted a research project on
providing close up access to slides for DeafBlind users a few years ago. We
found that users had different "visual maps" or areas of focus, and that
they preferred customized close up views of slides.

**Comment by JWJPaton** [comment]
Thanks @raja-kushalnagar-gallaudet,
Sightloss is always a broad spectrum and deafblindness adds a complex set of communication needs. I remember that in terms of signing to some deafblind people the signing space was kept smaller and closer to the lips. 
A quick google of USHER to remind me shows that it often includes a level of RP (retinitis pigmentosa) which creates blotches of dead spots in your vision (as well as poor sight in low light conditions). That would make sense that for static images users would find magnification easier but for signing they would want to position their eyes to avoid any of their dead spots covering the signing space. If you do research on the preferred sizing of sign language translation by deafblind signers I'd be interested in reading it. It may be tricky to get statistical significance though for such a diverse minority group.

**Comment by raja-kushalnagar-gallaudet** [comment]
You are right that people with Usher's syndrome have a wide range of visual
abilities which are progressively reduced. So visual abilities will vary
both by individual and over time, and that it will be difficult to get
meaningful statistical significance. I will discuss it with other
researchers, and send an update.
